webpack.config.js

```diff
@@ -15,15 +15,17 @@ module.exports = {
     filename: 'js/[name].js'
   },
   resolve: {
-    extensions: ['.js', '.vue', '.json'],
+    extensions: ['.js', '.vue', '.json', '.css', '.scss'],
```

Comment:
This change is not necessary. The Netlify build was failing because of an environment misconfiguration which is fixed now.